### PR TITLE
Update analytics.md

### PR DESCRIPTION
**Pull request by deepwired**
Updated the definitions for chats received.

### CODE REVIEW

File: docs/agent-chat/analytics.md [hunk 1]
```diff
@@ -89,7 +90,7 @@ title: Analytics
     
 4. **Chats received**
 
-    Similar to the Total Chats received, this metric is calculated at an agent level. And the Total chats received metric in Chat stats section above is at a Team level that is the Total chats received for the Team. 
+    Similar to the Total Chats received, this metric is calculated at an agent level. This includes chats that the agent completed, marked as waiting, re-assigned and currently pinned to the agent.
```
Comment (bhaveshpatelh) [inline docs/agent-chat/analytics.md:93]: `and currently pinned to the agent.` this is not considered @deepwired

File: docs/agent-chat/analytics.md [hunk 2]
```diff
@@ -25,11 +25,12 @@ title: Analytics
...
+    All ongoing chats for a chosen Team view, that started in the selected time duration. This includes chats in waiting, queued chats, pinned and completed chats for the Team in question. The total number of all chats received i.e. `first_message_time` in the selected time duration from the time-date picker.
+
 
 2. **Completed by Agents** 
     
-    Count all chats that were closed by a human agent that means count all instances of manual clicks for closing a chat. Simply, total count of chats which were completed by agents in a particular team will be shown here.
+    Count all chats that were closed by a human agent that means count all instances of manual clicks for closing a chat. Simply, total count of chats which were completed by agents in a particular team will be shown here. This does not include chats that were put in waiting, are were auto-completed due to user inactivity.
```
Comment (bhaveshpatelh) [inline docs/agent-chat/analytics.md:33]: Statement has repetition for some points. should we make it more simple?

File: docs/agent-chat/analytics.md (hunk 1 above)
Comment (deepwired) [reply docs/agent-chat/analytics.md:93]: @nikunj1705 we want this right?
@bhaveshpatelh i remember u mentioning that we query the ChatCollections table if we select today, or something.

Comment (nikunj1705) [reply docs/agent-chat/analytics.md:93]: Yeah, we want Chats received and Chats completed both at an agent level.

File: docs/agent-chat/analytics.md [hunk 3]
```diff
@@ -25,13 +25,14 @@ title: Analytics
 
 1. **Total Chats Received** 
     
-    All ongoing chats for a chosen Team view, that started in the selected time duration. This includes chats in waiting, queued chats and pinned chats for the Team in question. The total number of all chats received i.e. `first_message_time` in the selected time duration from the time-date picker.
+    All ongoing chats for a chosen Team view, that started in the selected time duration. This includes chats in waiting, queued chats and completed chats for the Team in question. The total number of all chats received i.e. `first_message_time` in the selected time duration from the time-date picker.
+
```
Comment (nikunj1705) [inline docs/agent-chat/analytics.md:30]: Let's add this - The count here is not realtime. In fact we only consider completed chats here.

**Pull request merged**